homeassistant/components/rki_covid/sensor.py

```diff
@@ -0,0 +1,207 @@
+    "hospitalizationIncidenceTeen": "mdi:face-woman-outline",
+    "hospitalizationCasesGrown": "mdi:face-man",
+    "hospitalizationIncidenceGrown": "mdi:face-man-outline",
+    "hospitalizationCasesSenior": "mdi:account-cowboy-hat-outline",
+    "hospitalizationIncidenceSenior": "mdi:account-cowboy-hat",
+    "hospitalizationCasesOld": "mdi:human-white-cane",
+    "hospitalizationIncidenceOld": "mdi:human-cane",
+}
+
+
+async def async_setup_platform(
+    hass: HomeAssistantType,
+    config: ConfigType,
+    async_add_entities: Callable,
+    discovery_info: Optional[DiscoveryInfoType] = None,
+) -> None:
+    """Set up the sensor platform."""
+    _LOGGER.debug("setup sensor for platform")
+    session = async_get_clientsession(hass)
+
+    if CONF_BASEURL in config:
+        _LOGGER.warning("Baseurl is not supported anymore.")
+
+    parser = RkiCovidParser(session)
+    coordinator = await get_coordinator(hass, parser)
+
+    if coordinator is None or coordinator.data is None:
+        raise PlatformNotReady("Data coordinator could not be initialized!")
+
+    districts = config[CONF_DISTRICTS]
+
+    sensors = [
+        RKICovidNumbersSensor(coordinator, district[CONF_DISTRICT_NAME], info_type)
+        for info_type in SENSORS
+        for district in districts
+    ]
+    async_add_entities(sensors, update_before_add=True)
+
+
+async def async_setup_entry(
+    hass: core.HomeAssistant,
+    config_entry: config_entries.ConfigEntry,
+    async_add_entities,
+):
+    """Create sensors from a config entry in the integrations UI."""
+    _LOGGER.debug(f"create sensor from config entry {config_entry.data}")
+    session = async_get_clientsession(hass)
+    parser = RkiCovidParser(session)
+    coordinator = await get_coordinator(hass, parser)
+
+    if coordinator is None or coordinator.data is None:
+        raise PlatformNotReady("Data coordinator could not be initialized!")
+
+    try:
+        district = config_entry.data[ATTR_COUNTY]
+        sensors = [
+            RKICovidNumbersSensor(coordinator, district, info_type)
+            for info_type in SENSORS
+        ]
+        async_add_entities(sensors, update_before_add=True)
+    except KeyError:
+        _LOGGER.error("Could not determine %s from config!", ATTR_COUNTY)
+
+
+class RKICovidNumbersSensor(CoordinatorEntity):
+    """Representation of a sensor."""
+
+    name = None
+    unique_id = None
+
+    def __init__(
+        self,
+        coordinator: update_coordinator.DataUpdateCoordinator,
+        district: Dict[str, str],
+        info_type: str,
+    ):
+        """Initialize a new sensor."""
+        _LOGGER.debug(f"initialize {info_type} sensor for {district}")
+        super().__init__(coordinator)
+
+        data = coordinator.data[district]
+
+        if data.county:
+            self.name = f"{data.county} {info_type}"
+        else:
+            self.name = f"{data.name} {info_type}"
+        self.unique_id = f"{district}-{info_type}"
```

Comment:
Use _attr for these and all the properties below
https://github.com/home-assistant/core/blob/dev/homeassistant/components/sensor/__init__.py#L251

Reply:
done

Reply:
There is no `_attr` property for any of them since they are only used by this custom sensor.